store/tikv/gcworker/gc_worker.go

```diff
@@ -425,6 +431,12 @@ func resolveLocks(ctx goctx.Context, store tikv.Storage, safePoint uint64, ident
 		default:
 		}
 
+		// We'd better keep 'req.ScanLock.StartKey' the same as 'key' to avoid
+		// the error 'key is not in region' when the region split for several pieces, or
```

Comment:
Region splitting and merging will cause region error, not `key is not in region`. Please fix this misunderstanding comment.

Reply:
ok